### PR TITLE
adding a makefile

### PR DESCRIPTION
**Pull request by asibille-indeed**
### Types of changes

- [ ] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- 
-

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+install: ##do all the npm commands to run the project
+	npm install
+	npm run build
+	upm link
+
+test: #run test suite 
+	npm test
+
+test_example: ##run test using the manifest file
```
Comment (moin-oss) [inline Makefile:9]: Need to add `cp sample-input.yml test.yml`, but looks good otherwise!

Comment (asibille-indeed) [reply Makefile:9]: I wasn't totally sure that people would want to cp over the example file every time they want to run it (eg: if they have made some changes in the sample file they're using)

Comment (asibille-indeed) [reply Makefile:9]: (But I do agree if you haven't already copied it over it would be a problem)

Comment (moin-oss) [reply Makefile:9]: Is there a way to check that one already exists, and only copy it if there isn't one?

Comment (asibille-indeed) [reply Makefile:9]: Yeah I can do that

**Review by moin-oss — APPROVED** [review]

**Pull request merged**